integration_tests/models/any_type_column_anomalies_no_timestamp.sql

```diff
@@ -0,0 +1,37 @@
+-- depends_on: {{ ref('any_type_column_anomalies_validation') }}
```

Comment:
This model has data for testing anomalies in columns as well. 
I think it's a small effort to add the column anomalies where there is no TS to e2e as well.